addons/event/static/src/scss/event.scss

```diff
@@ -3,25 +3,68 @@
     .o_kanban_record {
         position: relative;
         padding: 0;
-
         display: flex;
 
+        .o_kanban_record_title {
+            font-size: 1.2em;
+        }
+
         .o_event_left {
-            flex: 1 1 25%;
             text-align: center;
+            flex: 1 1 25%;
             color: white;
             background-color: $o-brand-primary;
             padding: 8px;
-
             .o_day {
                 font-size: 26px;
             }
         }
 
         .o_event_right {
-            flex: 1 1 75%;
+            display: flex;
+            flex-direction: column;
+            justify-content: space-between;
             padding: 8px;
             margin-left: 8px;
+            width: 70%;
+
+            .o_event_stat {
```

Comment:
with the existing CSS classes, I cannot place the elements exactly how we want (pixels align with the bottom/right)
cf: review remark on the pad

Reply:
@tde-banana-odoo I removed as many as I can lines in this file, but still need some elements because the bottom right elements is in "absolute position", and has to be aligned with the "excepted/confirmed" elements